code-of-conduct.md

```diff
@@ -21,13 +21,35 @@ Respectful behavior includes:
 * If you’re not sure, ask someone instead of assuming. No, really. Just ask the administrators. We’d rather hear from you than hear about something you said or did after the fact, and we are here to help.
 * Don’t be a bystander. Role model respectful behaviour, but also help to address disrespect when you see it.
 
-## Resolve Peacefully
+## When something happens
+1. Let the person know that what they did is not appropriate and ask them to stop and/or edit their message(s).
+2. That person should immediately stop the behavior and correct the issue.
+3. If this doesn’t happen, or if you’re uncomfortable speaking up, contact admins.
+4. As soon as available, an admin will join, identify themselves, and take further action (see below), starting with a warning, then temporary deactivation, then long-term deactivation.
```

Comment:
We'd probably want to find an easy, automated way to do temporary deactivations or perhaps just exclude it from the Code of Conduct if one isn't available. I'm not sure of a mechanism off the top of my head.

Reply:
oh is 'temporary deactivation' a paid feature?